README.md

```diff
@@ -22,9 +22,9 @@ wget https://raw.githubusercontent.com/frappe/bench/master/install_scripts/setup
 sudo bash setup_frappe.sh --setup-production
 ```
 
-This script should install the pre-requisites, install bench and setup an ERPNext site. Passwords for Frappe, Frappe Administrator and MariaDB (root) will be generated. You can then login as Administrator with the Administrator password printed.
+This script should install the pre-requisites, install bench and setup an ERPNext site. This will setup ERPNext with nginx, with Supervisor enabled and checkout the master branch of the ERPNext repo. Passwords for Frappe, Frappe Administrator and MariaDB (root) will be generated. You can then login as Administrator with the Administrator password printed. 
 
-If you want to develop ERPNext or any Frappe App, you can omit the "--setup-production" part from the command.
+If you want to develop ERPNext or any Frappe App, you can omit the "--setup-production" part from the command. This will setup ERPNext as well, however, it will run on NodeJS, have Supervisor disabled, and checkout the develop branch of the ERPNext repo. 
```

Comment:
Replace because `bench start` starts a bunch of processes and the node socketio server is one of them.